docs/Reference/Pantheon-API-Methods.md

```diff
@@ -1701,6 +2064,30 @@ Returns an array of [logs](../Using-Pantheon/Events-and-Logs.md) matching a spec
       } ]
     }
     ```
+    
+    ```bash tab="curl GraphQL request"
+    curl -X POST -H "Content-Type: application/json" --data '{"query": "{block(number:\"0x59fd\") {logs(filter:{topics:[[\"0xd3610b1c54575b7f4f0dc03d210b8ac55624ae007679b7a928a4f25a709331a8\", \"0x0000000000000000000000000000000000000000000000000000000000000002\"]]}) {index topics data account{address} transaction{hash} }}}"}' http://localhost:8547/graphql
```

Comment:
This should be a number not a hex string
```suggestion
    curl -X POST -H "Content-Type: application/json" --data '{"query": "{block(number:23037) {logs(filter:{topics:[[\"0xd3610b1c54575b7f4f0dc03d210b8ac55624ae007679b7a928a4f25a709331a8\", \"0x0000000000000000000000000000000000000000000000000000000000000002\"]]}) {index topics data account{address} transaction{hash} }}}"}' http://localhost:8547/graphql
```

But if we are matching the JSON-RPC calls I think it is

```suggestion
    curl -X POST -H "Content-Type: application/json" --data '{"query": "{blocks(from:0) {logs(filter:{topics:[[\"0xd3610b1c54575b7f4f0dc03d210b8ac55624ae007679b7a928a4f25a709331a8\", \"0x0000000000000000000000000000000000000000000000000000000000000002\"]]}) {index topics data account{address} transaction{hash} }}}"}' http://localhost:8547/graphql
```

However that might be a too-large query for the server based on how GraphQL queries.

Reply:
Addressed first comment.  Second comment will be addressed with https://pegasys1.atlassian.net/secure/RapidBoard.jspa?rapidView=38&view=planning&selectedIssue=PAN-2714